src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -1471,6 +1508,14 @@ theorem mul_twoPow_eq_shiftLeft (x : BitVec w) (i : Nat) :
       apply Nat.pow_dvd_pow 2 (by omega)
     simp [Nat.mul_mod, hpow]
 
+theorem twoPow_zero {w : Nat} : twoPow w 0 = 1#w := by
+  apply eq_of_toNat_eq
+  simp
+
+@[simp]
+theorem getLsb_one {w i : Nat} : (1#w).getLsb i = (decide (0 < w) && decide (0 = i)) := by
```

Comment:
This is a minor stylistic point, but I think it would look cleaner to write
`(1#w).getLsb i = decide (0 < w ∧ 0 = i)` instead of `(decide (0 < w) && decide (0 = i))`.
I guess it's a matter of preference, so it's up to you.

Reply:
@AtticusKuhn we write it this way because it's the simp normal form.